packages/babel-config/src/api.ts

```diff
@@ -137,9 +138,10 @@ export const getApiSideBabelPlugins = (
 
 export const getApiSideBabelConfigPath = () => {
   const p = path.join(getPaths().api.base, 'babel.config.js')
-  if (fs.existsSync(p)) {
+  if (existsSync(p)) {
     return p
   } else {
+    // Null and undefined do not have the same effect I believe
```

Comment:
"I believe" doesn't signal a lot of confidence in the code 😛 
Can you look it up/try it out and then modify the comment accordingly?

Reply:
Haha yes correct - low confidence. I may be misremembering but this  was possibly one of the odd undocumented behaviours (at the time) - I'll verify and remove the comment.